taxcalc/tests/test_calculate.py

```diff
@@ -12,19 +12,18 @@
 from taxcalc import *
 import copy
 
-WEIGHTS_FILENAME = "../../WEIGHTS_testing.csv"
-weights_path = os.path.join(CUR_PATH, WEIGHTS_FILENAME)
-weights = pd.read_csv(weights_path)
 
-all_cols = set()
-tax_dta_path = os.path.join(CUR_PATH, "../../tax_all1991_puf.gz")
-tax_dta = pd.read_csv(tax_dta_path, compression='gzip')
-
-# Fix-up. MIdR needs to be type int64 to match PUF
-tax_dta['midr'] = tax_dta['midr'].astype('int64')
-tax_dta['s006'] = np.arange(0, len(tax_dta['s006']))
-
-irates = {1991: 0.015, 1992: 0.020, 1993: 0.022, 1994: 0.020, 1995: 0.021,
+# use 1991 PUF to emulate private 2009 PUF
```

Comment:
to avoid confusion, I would say "use simulated 1991 PUF", since the dataset is really some modified version of 1991 data that was coerced into something resembling the PUF format.